http.echo.jwt/README.md

```diff
@@ -35,7 +35,7 @@ jwt encode \
 ```
 The signed JWT token is shown below.
 ```
-eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6ImV4YW1wbGUifQ.eyJhdWQiOiJodHRwczovL2FwaS5leGFtcGxlLmNvbSIsImV4cCI6MTk2ODg2Mjc0NywiaXNzIjoiaHR0cHM6Ly9hdXRoLmV4YW1wbGUuY29tIiwic2NvcGUiOiJlY2hvOnN0cmVhbSJ9.glfCpnhVkQFf5zXlSFDWYsHyFhFEuxmRXVu8AbFXh67FzcjwzEcMgw1Zt7_SETyXHpNl1HhOgLohaVCkGxVG2iiOq0MJO00_l6X125itdY37noOFiGWTHb8uosGI4V3NhhCKyoVLtl3b9X4c6pCxHoQo7XkT1xmcjSeCKQenXpuX5WnKMIZsyBxUsOxg1cv3K7mg6WnKOlXWGjvCAoomUjIGiGDruFQMP1UzniMgY0b0IrofijiNB3HEKQQcU44MD7jH9lldrea1vaKnxYwmiaq7g7RsYMFXeNLzWz6hY61ColSeEUCiDtpVSNCyjKZHkuLA7yLQ-pvipwCpT0jU1Q
+eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6ImV4YW1wbGUifQ.eyJhdWQiOiJodHRwczovL2FwaS5leGFtcGxlLmNvbSIsImV4cCI6MTk2ODg2Mjc0NywiaXNzIjoiaHR0cHM6Ly9hdXRoLmV4YW1wbGUuY29tIn0.k4Aq93RzFpOBBwuUEewJUq1Wj1F0csfW4c_eGaQY9xNk8WC1C_rhmWkiprApkoVoaUJI7PVemUFfwKmx3XVWTYB3AQUihGGDKA6TRN2kTfkd1Vm_tBbn6a1nsUKbfl70vFD51jebJ9w5yG2b_jEiqtt6eOW99KNNRdAi5U0z7NHHIniu8Yfi5qrK0IBJBBWOoe-D-539ZzWWlMZKA5n1BJZ6x5ZOJAbYWdoMxr73uo7p9rWdVNk-61KsqVSSkCy92dq_d0Uoa3Q8xT5cwpWoljDwl-jB1O6PBwR7MVGJfihFfQVimt0NDnWi8TPXyBUq7RWGwfmQdEHwcrGAnJiaNg
```

Comment:
What problem does this change address?

Reply:
Line 27 creates a token without `echo:stream` scope and line 43 actually uses that. The token displayed here at line 38 should be this one. However, it wasn't, it was the other token, the one **with** `echo:stream` scope (line 55, 67, 72).